rskj-core/src/main/java/org/ethereum/rpc/LogFilter.java

```diff
@@ -18,54 +18,46 @@
 
 package org.ethereum.rpc;
 
-import java.util.Collection;
-import java.util.List;
-
-import co.rsk.rpc.netty.ExecTimeoutContext;
-import org.ethereum.core.Block;
-import org.ethereum.core.Blockchain;
-import org.ethereum.core.Bloom;
-import org.ethereum.core.Transaction;
-import org.ethereum.core.TransactionReceipt;
-import org.ethereum.db.TransactionInfo;
-import org.ethereum.rpc.exception.RskJsonRpcRequestException;
-import org.ethereum.vm.LogInfo;
-
 import co.rsk.core.RskAddress;
 import co.rsk.crypto.Keccak256;
+import co.rsk.jsonrpc.JsonRpcError;
 import co.rsk.logfilter.BlocksBloom;
 import co.rsk.logfilter.BlocksBloomStore;
+import co.rsk.rpc.netty.ExecTimeoutContext;
 import co.rsk.util.HexUtils;
+import org.ethereum.core.*;
+import org.ethereum.db.TransactionInfo;
+import org.ethereum.rpc.exception.RskJsonRpcRequestException;
+import org.ethereum.vm.LogInfo;
 
 import javax.annotation.Nullable;
+import java.util.Collection;
+import java.util.List;
 
 /**
  * Created by ajlopez on 17/01/2018.
  */
 public class LogFilter extends Filter {
-    class LogFilterEvent extends FilterEvent {
-        private final LogFilterElement el;
-
-        LogFilterEvent(LogFilterElement el) {
-            this.el = el;
-        }
 
-        @Override
-        public LogFilterElement getJsonEventObject() {
-            return el;
-        }
-    }
-
-    private AddressesTopicsFilter addressesTopicsFilter;
-    private boolean fromLatestBlock;
-    private boolean toLatestBlock;
+    private final AddressesTopicsFilter addressesTopicsFilter;
+    private final boolean fromLatestBlock;
+    private final boolean toLatestBlock;
     private final Blockchain blockchain;
+    private final long maxBlocksToQuery;
+    private final long maxLogsToReturn;
 
+    @Deprecated
     public LogFilter(AddressesTopicsFilter addressesTopicsFilter, Blockchain blockchain, boolean fromLatestBlock, boolean toLatestBlock) {
+        this(addressesTopicsFilter, blockchain, fromLatestBlock, toLatestBlock, 0, 0);
+    }

```

Comment:
can we remove this deprecated one? Seems we could use the other one with zeros being provided for `maxBlocksToQuery` and `maxLogsToReturn`?

Reply:
yep, just preferred to deprecated it before delete. I'll do it